pkg/config/validation/validation.go

```diff
@@ -598,6 +599,16 @@ func validateTLSOptions(tls *networking.ServerTLSSettings) (v Validation) {
 		return
 	}
 
+	if tls.ServerCertificate != "" {
```

Comment:
This is actually a valid setup and people like us use it. Is there a specific reason why we want to introduce this new warning now?

Reply:
Feel the same if the file mounted cert feature is supported forever, why warn it

Reply:
Users get confused that they *should use* it, when 99% of users should not. And most other ingress do not have this either, so folks get mislead into thinking they need to *migrate* to this model. That is my impression anyhow. Our docs do not have filemounted referenecs anymore at all (except in reference)

Reply:
AFAIK, sds could replace file mounted certs completely. And we should suggest changing to use it by changing the api doc.